src/releasers/go-yoshi.ts

```diff
@@ -166,7 +166,7 @@ export class GoYoshi extends ReleasePR {
       changelogEntry,
       updates,
       version: candidate.version,
-      includePackageName: false,
+      includePackageName: this.monorepoTags,
```

Comment:
This logic is the same as all of the other releasers now, we include the package name if the release is for a submodule.